docs/vault/guide.md

```diff
@@ -113,11 +113,69 @@ To mint with your own (or another specific) Vault, you can use the Polkadot.js i
 
 You Vault only starts earning rewards once BTC is locked - and rewards are determined by your share of the total BTC locked in the system. Hence, you can increase your rewards by bringing your own BTC into the system. 
 
-## Dashboard
+## Monitoring
+
+Vault operators can monitor their clients using both the [Dapp Vault Dashboard](vault/guide?id=dapp-vault-dashboard) and [Prometheus / Grafana](vault/guide?id=prometheus-and-grafana).
+
+### Dapp Vault Dashboard
+
+You can monitor the operation of your Vault on the Vault dashboard in the Dapp, by adding the key to the [polkadot{.js} extension](https://polkadot.js.org/extension/).
+
+Once the Vault is up and running, a "Vault" tab will appear in the sidebar of the Dapp at [testnet.interlay.io](https://testnet.interlay.io/) (or you can access directly at [testnet.interlay.io/vault](https://testnet.interlay.io/vault)).
+
+### Prometheus and Grafana
+
+Similar to how Substrate provides functionality for [monitoring Nodes](https://docs.substrate.io/tutorials/v3/node-metrics/), Vault operators have the option to monitor their clients using Prometheus and Grafana. The key used for tracking the metrics is comprised of the collateral and wrapped currencies of the client (e.g. "KSM_KBTC") - this allows for tracking clients with different currency combinations in Grafana.
+
+### Metrics
```

Comment:
Better to link to a list of metrics in the clients repo so we don't have to update this list. I would also just include the query against prometheus port that displays the available metrics like `curl -v IP:9615/metrics`